meta/data.go

```diff
@@ -34,6 +36,18 @@ type Data struct {
 	MaxNodeID       uint64
 	MaxShardGroupID uint64
 	MaxShardID      uint64
+
+	Logger *log.Logger
```

Comment:
There is no need for this now. `dataLogger` is now global so you can just reference it directly in the code.